DataConnectors/Connector_syslog_WatchGuardFirebox.json

```diff
@@ -0,0 +1,117 @@
+{
+    "id": "WatchguardFirebox",
+    "title": "Firebox",
+    "publisher": "Watchguard",
+    "descriptionMarkdown": "Syslog is an event logging protocol that is common to Linux. Applications will send messages that may be stored on the local machine or delivered to a Syslog collector. When the Agent for Linux is installed, it configures the local Syslog daemon to forward messages to the agent. The agent then sends the message to the workspace.\n\n[Learn more >](https://aka.ms/sysLogInfo)",
+    "additionalRequirementBanner": "This data connector depends on a parser based on Kusto Function to work as expected. Follow the steps to use this Kusto Function alias **<enter the Kusto Function alias>** in queries and workbooks. [Follow steps to get this Kusto Function>](<Link to Kusto Function on Azure Sentinel GitHub>)  ",
+    "graphQueries": [
+        {
+            "metricName": "Total data received",
+            "legend": "Watchguard",
+            "baseQuery": "Firebox"

```

Comment:
"baseQuery": "DATATYPE_NAME", Validate DATATYPE_NAME represent both provider, appliance name and type of data [optionally] as a short name in the data type name.

DATATYPE_NAME should be WatchGuardFirebox